Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -93,22 +102,37 @@ public final class ExploreView: UIView {
                     let cell = collectionView.dequeue(ExploreBrazeBannerCell.self, for: indexPath)
                     cell.configure(banner: viewModel.brazePromo)
                     return cell
+                case .recommendation(let viewModel):
+                    let cell = collectionView.dequeue(StandardAdRecommendationCell.self, for: indexPath)
+                    cell.imageDataSource = self
+                    cell.configure(with: viewModel, atIndex: indexPath.row)
+                    cell.index = indexPath.row
```

Comment:
The method `configure(with:atIndex:)` sets the `index` property when called, so the `cell.index …` line can be removed.